### PR TITLE
Limit number of characters in comments

### PR DESCRIPTION
**Pull request by hedudelgado**
@alexnoox

### CODE REVIEW

**Comment by alexnoox** [comment]
Can you add a # of characters left?

This is twitter:
screen shot 2016-12-19 at 11 16 54 am

**Comment by hedudelgado** [comment]
@alexnoox

**Comment by hedudelgado** [comment]
@alexnoox  changed it.

**Comment by hedudelgado** [comment]
@alexnoox changed it.

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 1]
```diff
@@ -8,11 +8,13 @@
     <div class="pull-right">
       <star-rating ng-model="vm.appRating"></star-rating><br/>
     </div>
-    <textarea class="form-control" rows="5" ng-model="vm.modal.model.description" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.show.tell_others' | translate}}" id="comment"></textarea>
+    <textarea class="form-control" rows="5" ng-trim="false" maxlength="{{vm.commentMaxLenght}}" ng-model="vm.modal.model.description" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.show.tell_others' | translate}}" id="comment"></textarea>
+
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Please remove this empty line.

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
       <star-rating ng-model="vm.appRating"></star-rating><br/>
     </div>
-    <textarea class="form-control" rows="5" ng-model="vm.modal.model.description" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.show.tell_others' | translate}}" id="comment"></textarea>
+    <textarea class="form-control" rows="5" ng-trim="false" maxlength="{{vm.commentMaxLenght}}" ng-model="vm.modal.model.description" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.show.tell_others' | translate}}" id="comment"></textarea>
+    
   </div>
   <div class="modal-footer">
     <div class="row">
       <div class="col-sm-12">
+      <span>{{vm.commentMaxLenght - vm.modal.model.description.length}}</span>
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Please fix the indentation.

**Comment by hedudelgado** [comment]
Done @alexnoox . Thanks

**Pull request merged**